apps/android_deploy/compile-models/README.md

```diff
@@ -0,0 +1,39 @@
+# TVM Compiled Models for Android Phone using Darknet frontend
```

Comment:
compile-models -> scripts

README.md 
Example script to prepare the pre-trained models